bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -397,7 +417,7 @@ async function addRecipient({ client, dataStore, integrationParameters, actionPa
                 {
                     email: actionParameters.recipient_Email ?? actionParameters.recipient_Email_Unknown,
                     name: actionParameters.recipient_Name,
-                    recipientId: Date.now(),
+                    recipientId: `1${moment().format('Hms')}`,
```

Comment:
I don't understand this.
Should it be some kind of "random" number?

Reply:
With `Date.now()` we were facing the as the value was getting out of the range for the `recipeintId` parameter.
Now we have used the moment js to get the `Hms` where `H = Hours, m=minutes, s= seconds` of the current time.
For more detail please refer to the link [here](https://momentjs.com/docs/#/displaying/)

Reply:
Yes, I know how moment format works. 
But I'm asking why are you generate some id which will be the same every 24 hours. And if it is ok that it can be the same.

Reply:
We need to have the unique`recipientId` within a single envelope. So there is less chance that a user will add two users at the same timestamp.

Reply:
If you think that it isn't a problem, then it is ok :)

Reply:
@Ajcitrite please use `const uud = library.load('uuid')` and then `recipientId: uuid.v4()`